### PR TITLE
issues and pr templates and labeler workflow

### PR DESCRIPTION
**Pull request by Skenvy**
Addresses #2405 
Adds the ISSUE_TEMPLATE's, the PR template, and a issue labeler with config targetting the bug-report format that will generate an issue body that includes;
```md

### Operating System

some_choice_from_the_dropdown

```

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  validations:
+    required: true
+- type: textarea
+  id: stacktrace
+  attributes:
+    label: If you have a stack trace, please add it here.
+- type: textarea
+  id: additional
+  attributes:
+    label: Anything else you feel useful to add?
```
Comment (hajimehoshi) [inline .github/ISSUE_TEMPLATE/bug-report.yml:60]: Add a new line.

**Review by hajimehoshi — APPROVED** [review]
LGTM, thanks!

File: .github/workflows/issue-labeler.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    types: [opened, edited]
+
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: github/issue-labeler@6ca237433dbbb8e475241b7f38f4600d9e296c57 # v2.5
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
+        configuration-path: .github/issue-labeler-config.yml
```
Comment (hajimehoshi) [inline .github/workflows/issue-labeler.yml:13]: Can we move `.github/issue-labeler-config.yml` to `.github/workflows/issue-labeler-config.yml`?

**Review by hajimehoshi — COMMENTED** [review]
Sorry, I have one more request:

File: .github/workflows/issue-labeler.yml (hunk 2 above)
Comment (Skenvy) [reply .github/workflows/issue-labeler.yml]: Moved it to `.github/workflows/issue-labeler/config.yml`

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<!--
+Thanks for sending a pull request!
+If this is your first time, please read the contributor guidelines:
+https://github.com/hajimehoshi/ebiten/blob/main/CONTRIBUTING.md
+Also please adhere to our Code of Conduct:
+https://github.com/hajimehoshi/ebiten/blob/main/CODE_OF_CONDUCT.md
+-->
+# What issue is this addressing?
+<!-- Fixes #<issue number> -->
```
Comment (hajimehoshi) [inline .github/PULL_REQUEST_TEMPLATE.md:9]: I prefer `Closes` to `Fixes` (I can fix this later if you are busy)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<!--
+Thanks for sending a pull request!
+If this is your first time, please read the contributor guidelines:
+https://github.com/hajimehoshi/ebiten/blob/main/CONTRIBUTING.md
+Also please adhere to our Code of Conduct:
+https://github.com/hajimehoshi/ebiten/blob/main/CODE_OF_CONDUCT.md
+-->
```
Comment (hajimehoshi) [inline .github/PULL_REQUEST_TEMPLATE.md:7]: I was wondering why these are HTML comments. I can fix them later anyway...

Comment (hajimehoshi) [reply .github/PULL_REQUEST_TEMPLATE.md:7]: No problem, I understand why.

**Pull request merged**